tests/integration/snapshot/member_test.go

```diff
@@ -54,7 +54,7 @@ func TestSnapshotV3RestoreMultiMemberAdd(t *testing.T) {
 	}
 	defer cli.Close()
 
-	urls := newEmbedURLs(2)
+	urls := newEmbedURLs(t, 2)
```

Comment:
i dont understand why Github is showing this file as changed.
this is my local commit
```
$ git show --name-only 
commit e5216c146b18eb1d92bd75780b7bd1ae14924fc2 (HEAD -> e2e-txn-test, origin/e2e-txn-test)
Merge: 82a258dc3 24002fb09
Author: Vimal Kumar <vimal78@gmail.com>
Date:   Wed May 11 02:07:04 2022 +0530

    tests: Migrate Txn tests to common functional test framework

etcdctl/ctlv3/command/txn_command.go
etcdctl/ctlv3/command/util.go
etcdctl/ctlv3/command/watch_command.go
go.mod
tests/common/txn_test.go
tests/e2e/ctl_v3_txn_test.go
tests/framework/config/client.go
tests/framework/e2e/e2e_test.go
tests/framework/e2e/etcdctl.go
tests/framework/integration.go
tests/framework/interface.go
tests/go.mod
tests/go.sum
```

Reply:
same for files
```
tests/framework/integration/cluster.go
tests/integration/snapshot/v3_snapshot_test.go
```

Reply:
after rebase to main, these files are not changed